src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/sort/SortClause.java

```diff
@@ -85,7 +85,26 @@ public boolean hasVectorizeSearchClause() {
             .equals(DocumentConstants.Fields.VECTOR_EMBEDDING_TEXT_FIELD);
   }
 
+  public SortExpression bm25SearchExpression() {
+    if ((sortExpressions == null) || sortExpressions.size() != 1) {
```

Comment:
Deserializer enforces limitation such that bm25 sort can only exists as the one sort expression -- fails if there are others. So at point this method may be called, invariant already checked.